src/common/components/nav/nav.component.js

```diff
@@ -39,10 +39,22 @@ class NavController{
     this.sessionModalService = sessionModalService;
 
     this.imgDomain = envService.read('imgDomain');
-    this.navFeed = '/bin/cru/site-nav.json';
   }
 
   $onInit() {
+    //init attributes
```

Comment:
That's correct.  It is correct at the end of constructor, then gets set to undefined before $onInit.  I guess I'll leave it how I have it unless you have another suggestion.